calendar_switzerland/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Calendar Switzerland
+======================
+Add custom functionality for Switzerland.
```

Comment:
Can you just add a small sentence explaining the email modification you are doing (send invitation e-mails only to employees)

Reply:
Done.